src/cljc/quil/core.cljc

```diff
@@ -819,7 +821,8 @@
   "Extracts the blue value from a color, scaled to match current color-mode.
   Returns a float."
   [color]
-  (.blue (current-graphics) (unchecked-int color)))
+  #?(:clj (.blue (current-graphics) (unchecked-int color))
+     :cljs (.blue (current-graphics) color)))
```

Comment:
Does using unchecked-int causes now some problems in cljs?

Reply:
functions expect p5.Color|Number[]|String https://github.com/processing/p5.js/blob/master/src/color/creating_reading.js#L346

```
green() was expecting p5.Color|Number[]|String for parameter #0 (zero-based index), received number instead
```

Alternative is to remove `unchecked-int` for both. Do you know what the purpose of `unchecked-int` is there?

Reply:
Ah, so p5js no longer encodes color as number. 

I think `unchecked-int` was added to make sure that we convert whatever argument we got into int. For example user could pass clojure fraction or bigint and java API takes only int. Hm, actually thinking about it you probably right. When int represents color - it's not really a number, but a bit-encoding of 4 values in single number. So conversion from other type (e.g. bigint) to int doesn't make sense as it will not have correct bits anyway. 

So yes, removing unchecked-int conversions from functions that expect color sounds good. Do you want to do it in this pull request or follow up?

Reply:
Rewrote the commits so we get a clean history in the quil repo. Did you see the last message below regarding some of the tests looking different?

Reply:
You can ignore my last comment regarding differences in HSB tests. In [p5 colorMode](https://p5js.org/reference/#/p5/colorMode) the default for HSB matches [processing colorMode](https://processing.org/reference/colorMode_.html) i.e. `colorMode(HSB, 360, 100, 100, 1)`. I'm guessing it was 255 in processing.js because if I try 255 the tests look the same.